.github/workflows/stale.yml

```diff
@@ -0,0 +1,22 @@
+name: Close and mark stale issue
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+    steps:
+      - uses: actions/stale@v3
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          stale-issue-message: "Oops, seems like we needed more information for this issue, please comment with more details or this issue will be closed in 7 days."
+          close-issue-message: "This issue was closed because it is missing author input."
+          stale-issue-label: "stale"
+          any-of-labels: "need-author-input"
+          days-before-issue-stale: 6
```

Comment:
Is 6x days enough to mark things as stale?

Could you please clarify the criteria by which it happens?

For example, will it apply only to issues created by non-code owners?

Reply:
Could you add this to the PR description?

Reply:
@maschad Perfect, thanks for clarifying.

I think 6x days may not be enough, can we start with 14 days and see how it goes?

Reply:
Sure